sms_sponsorship/models/sms_child_request.py

```diff
@@ -195,6 +209,9 @@ def onchange_partner_id(self):
     def change_child(self):
         """ Release current child and take another."""
         self.hold_id.write({"sms_request_id": False})
+        if self.type == "sms":
+            self.child_id.write({"state": "N"})
+        self.child_id.add_to_wordpress()
```

Comment:
This has to be moved in `compassion-switzerland` repo as this module has no link with wordpress website. Plus I'm not sure of the part only for "SMS", wouldn't it be regardless of the type of request?